birdhouse/optional-components/test-cowbird-jupyter-access/docker-compose-extra.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.4"
+services:
+  test-cowbird-jupyter-access:
+    image: pavics/workflow-tests:210216
```

Comment:
I had just copied the image version that was found in the `env.local.example`, which was 210216, but it would make more sense to get the latest. It shouldn't be a problem, I will update this.

Reply:
That is my intention (using the one tagged by birdhouse-deploy).
I think this was applied because our DACCS-config for the Jenkins CI running the PRs used to hardcode `pavics/workflow-tests:230601` with extra EO and NLP images from https://github.com/crim-ca/pavics-jupyter-images.
I pushed a fix yesterday that applies the upstream version from birdhouse-deploy to always apply what is currently active.

Reply:
Changed to use the latest image from the config.